tensorboard/webapp/feature_flag/force_svg_data_source.ts

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Injectable} from '@angular/core';
+import {FORCE_SVG_RENDERER} from '../webapp_data_source/tb_feature_flag_data_source_types';
```

Comment:
Rather than sharing the query parameter key perhaps we should use a name more like the one used by PersistentSettingsDataSource:

https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/persistent_settings/_data_source/persistent_settings_data_source.ts;l=22;drc=b0f27571ce3e6ce322b9f1b91a4e8773f9ca47e0

Prefixed with '\_tb\_'.

Reply:
That makes more sense. Changed.